hackmcgill/dashboard
# Feature/855 update settings dashboard

### Tickets:

fixes #855

### List of changes:

- Create settings page which allows admins to change the open, close, and confirm times for hackers
- Add functions into util file

### Type of change:

Please delete options that aren't relevant.


### How did you do this?

### How to test:

- Open dashboard, navigate to settings page
- Change the date / time
- Confirm on reload that the new date / time is returned by the api 

### Questions:

- I think this will fail if there does not exist a settings schema in the DB. I will need to check this, but if you want, please confirm.

### PR Checklist:


### Screenshots:

Pretty bare-bones. Not looking to make it nice atm, just getting it functional.
![image](https://user-images.githubusercontent.com/17463268/89115045-4d260c00-d451-11ea-890d-944346b1fabd.png)

Confirmation modal:
[![Image from Gyazo](https://i.gyazo.com/dd847c8058308f51b3f72b6f6416e9dc.gif)](https://gyazo.com/dd847c8058308f51b3f72b6f6416e9dc)
